customer-config/scan-and-go/sng.device-api-settings.v1.json

```diff
@@ -10,6 +10,12 @@
       "minimum": 0,
       "maximum": 100,
       "description": "The minimum battery level required to release a handheld scanner."
+    },
+    "scannerInactiveDaysBeforeRemoval": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 180,
+      "description": "The amount of days after which inactive handheld scanner is removed. Default value 0: scanners won't be removed."
```

Comment:
Number of days after which an inactive handheld scanner is removed. The default is set to 90 days.